examples/01-Getting-started.ipynb

```diff
@@ -0,0 +1,1012 @@
+{
```

Comment:
<p>I think we changed the notebook to use only movieid, userid, ratings (so inly rating.dat).</p><p> </p><p>I think you explained this in the above text</p><p> </p><pre class="ql-syntax" spellcheck="false">The original dataset has been preprocessed to make it easier to work with. Instead of having to deal with dat files, we can read files directly into cudf using the parquet format. 

The data has already been split for us into train and validation sets.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/NVTabular/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>good catch! removed incorrect information</p>